.github/workflows/main.yml

```
@@ -26,14 +26,14 @@ jobs:
       matrix:
         os: ['ubuntu-latest']
         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
-        cratedb-version: ['4.8.4', '5.3.2']
+        cratedb-version: ['4.8.4', '5.4.4']
```

Same here, lets use `5.4.5`?
```suggestion
        cratedb-version: ['4.8.4', '5.4.5']
```

---

Is it only on macOS?
```
$ docker pull crate:5.4.5
Error response from daemon: manifest for crate:5.4.5 not found: manifest unknown: manifest unknown
```

---

Well that tarball link works, so it will also work here to bump to 5.4.5. Thanks.

https://cdn.crate.io/downloads/releases/crate-5.4.5.tar.gz

---

https://github.com/docker-library/official-images/pull/15666 is still open